substrafl_examples/strategies_examples/plot_substrafl_fedavg.py

```diff
@@ -62,6 +62,10 @@
 
 N_CLIENTS = 2
 
+# Choose the subprocess mode to locally simulate the FL process
```

Comment:
please do not update this, it is already fixed in another PR: https://github.com/Substra/substra-documentation/pull/178

Reply:
done